jme3-core/src/main/java/com/jme3/texture/Image.java

```diff
@@ -253,77 +253,238 @@
          * Requires {@link Caps#SharedExponentTexture}.
          */
         RGB9E5(32,true),
+
+        @Deprecated
+        Reserved11(0),

```

Comment:
Adding this here changes the ordinals for the formats, which would break serialization (unfortunately, it uses the enum ordinal...)
New formats always have to be added at the bottom, and any existing formats can only be renamed, not removed